nodeJS/express/introduction_to_express.md

```diff
@@ -0,0 +1,98 @@
+### Introduction
+
+In the previous section, you got up and running with Node. You learned how to read and write data from files on the server, allowing you to serve up a multi-page website. You may have found this process somewhat verbose and cumbersome, and may have even wondered how on Earth we would manage more complex use cases.
+
+In this course, we will be using a backend framework called [Express](https://expressjs.com/), which will handle many of the implementation details for us. Express itself is an intentionally barebones and unopinionated framework; it allows us to do many things how we want, and to extend it with only the features we need. However, while this gives us great flexibility in how we do things, it can be a little tricky deciding between multiple viable options at times.
+
+Going forward, we will be diving into how we can use Express in various ways, such as to create a full-stack application using the Model View Controller (MVC) pattern, as well as to create a REST API just like ones you will have used before for things such as the Weather App or the React Shopping Cart projects. Along the way, we will be learning about things like databases, authentication, and much more. There is a lot to take in, so take it steady, and do not be afraid to ask our community for help in [our Discord server](https://discord.gg/theodinproject)!
```

Comment:
I'm not sure if this sentence belongs here, makes the lesson sound like an intro to a course. Don't feel strongly towards it, your call.

```suggestion
Going forward, we will be diving into how we can use Express in various ways, such as to create a full-stack application using the Model View Controller (MVC) pattern, as well as to create a REST API just like ones you will have used before for things such as the Weather App or the React Shopping Cart projects. There is a lot to take in, so take it steady, and do not be afraid to ask our community for help in [our Discord server](https://discord.gg/theodinproject)!
```